### PR TITLE
Bump distroless-iptables to v0.3.1 and Go to 1.21

### PR DESCRIPTION
**Pull request by jingyuanliang**

### CODE REVIEW

File: scripts/Dockerfile.in [hunk 1]
```diff
@@ -63,7 +63,7 @@ RUN /alpine-extractor.sh musl openssl-dev openssl-libs-static linux-headers
 
 ## The bash container fetches bash-static (to be used as bash) from Debian repo.
 
-FROM debian:11.4 AS bash
+FROM debian:bookworm-slim AS bash
```
Comment (aojea) [inline scripts/Dockerfile.in:66]: we had some hiccups with this change, have you verified this is ok?

Comment (jingyuanliang) [reply scripts/Dockerfile.in:66]: Tested in a cluster and confirmed it working.

**Review by MrHohn — APPROVED** [review]
/lgtm

**Pull request merged**